Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /app
 
 RUN apk update
 RUN apk upgrade --available
-RUN apk add libc6-compat openssl-dev build-base libpq-dev nodejs=~18 npm git
+RUN apk add libc6-compat openssl-dev build-base libpq-dev nodejs=~18 npm git python3
```

Comment:
Had to add this because accessible autocomplete uses node-sass no dart-sass, which is sad.